roles/ceph-common/handlers/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: update apt cache
+  apt: update-cache=yes
+
+- name: restart ceph mons
```

Comment:
Can you use the service module here to avoid using separate handlers per distribution?